lighthouse-core/audits/accessibility/axe-audit.js

```diff
@@ -77,9 +77,11 @@ class AxeAudit extends Audit {
       items = rule.nodes.map(node => ({
         node: /** @type {LH.Audit.Details.NodeValue} */ ({
           type: 'node',
+          // This selector represents the aXe css selectors
+          // Node selector can be accessed with node.selector
           selector: Array.isArray(node.target) ? node.target.join(' ') : '',
```

Comment:
Not 100% about this. I assumed that NodeValue selector should be the actual node's selector. But, this originally uses the aXe audit's css selectors (given by `node.target`), which might just be an audit exception?

Reply:
I think it's ok to keep this the way it is. @connorjclark are we worried about the difference between the selector returned by aXe and the selector we construct in `getNodeSelector()`?

Reply:
@paulirish didn't you look at the difference here between axe and us?

Reply:
@adrianaixba and I just deep dived into .target and what it's useful for. It addresses cases where the node is within shadowDOM or iframes. (because css selectors can't express that.)

Since Lighthouse can't do anything useful here (and our own getSelector doesnt handle the iframe/shadow DOM cases) I think we should ignore it.

So yeah let's use our own NodeValue selector here and discard their `target` selector array.